### ISSUE TITLE
UX Question

### ISSUE THREAD

**Issue by muffinjets** [MrRar/gps_locker#2]
Hey, thought I'd give this a try, but I'm a little confused on what the buttons mean

If the "Lock GPS" switch is on, does that mean my GPS is disabled?

Is the "Disable GPS" button only there for devices who can't enable/disable from the pull-down menu?

Thanks!

**Comment by MrRar** [comment]
The lock GPS switch when activated basically tells the app to try to lock the GPS. It might fail to do so for various reasons for example if the GPS is disabled on the device or if the signal is bad.

The disable GPS button is just a shortcut to open up GPS settings in the settings app. There is an option in there to enable/disable the GPS. If the GPS is disabled on the device the button says "Enable GPS"

**Comment by kaoneko** [comment]
What does "to lock the GPS" mean?

**Comment by muffinjets** [comment]
Yeah, I had the GPS 'locked' and then tried to see if Google Maps/OSMAnd could trace my position

If I'm misunderstanding the purpose of the app, then that's on me

**Comment by MrRar** [comment]
> What does "to lock the GPS" mean?

It means to put the GPS in a state where it provides location information. http://gpsinformation.net/main/slowlock.htm

> Yeah, I had the GPS 'locked' and then tried to see if Google Maps/OSMAnd could trace my position, and I was able to navigate to the next town over just the same.

I'm guessing you thought the app would cause the GPS to always report the same location while locked. Actually the app just tries to keep the GPS in a locked state. Meaning GPS is providing location information. For example the app could allow you to start using a navigation app more quickly if the navigation app does not need to wait to lock the GPS if it's already locked.
